src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
```

Comment:
Consider moving the Random instance to a class-level field instead of creating a new instance every time the `getRandomBall` method is called. This will be more efficient as you will reuse the same Random instance.